newsletter-january-2023.rst

```diff
@@ -0,0 +1,118 @@
+  We are aligning these two approaches,
+  making it easier to design and configure builds for a project's individual needs.
+- Continuing to restructure and rewrite the user documentation of Read the Docs.
+  We look forward to publishing the restructured documentation which is likely happening at the end of Q1.
+- Dashboard redesign: The entire Dashboard will be redesigned and re-implemented.
+  This work is about to be launched!
+
+
+Possible issues
+---------------
+
+Sphinx users should be aware that Sphinx 6.0 and 6.1 were released very recently.
+If you haven't pinned dependencies,
+the update might come automatically.
+If you choose to update,
+there are a few things to be aware of.
+Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+
+..
+  Tip of the month
+  ----------------
+  
+  TBD: Insert twitter embed
+
+
+Awesome Project of the month
+----------------------------
+
+`GeoPandas <https://geopandas.org/>`_ is an open source project for working with geospatial data which requires a lot of exciting features from the documentation - maps, plots etc. We especially like how the documentation is structured. See all the highlights in the following `Twitter thread <https://twitter.com/readthedocs/status/1603095976117522433>`_:
+
+.. raw:: html
+
+   <blockquote class="twitter-tweet"><p lang="en" dir="ltr">GeoPandas is an open source project to make working with <a href="https://twitter.com/hashtag/geospatial?src=hash&amp;ref_src=twsrc%5Etfw">#geospatial</a> data in <a href="https://twitter.com/hashtag/Python?src=hash&amp;ref_src=twsrc%5Etfw">#Python</a> easier. <a href="https://twitter.com/geopandas?ref_src=twsrc%5Etfw">@GeoPandas</a> extends the datatypes used by pandas to allow spatial operations on geometric types.<br><br>We want to highlight some things we love from their docs.<br><br>🤏 (small) 🧵 <a href="https://t.co/Hj82s6SDQP">pic.twitter.com/Hj82s6SDQP</a></p>&mdash; Read the Docs (@readthedocs) <a href="https://twitter.com/readthedocs/status/1603095976117522433?ref_src=twsrc%5Etfw">December 14, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Awesome Read the Docs Projects List 🕶️
+--------------------------------------
+
+Looking for more inspiration?
+Check out our new list `Awesome Read the Docs Projects <https://github.com/readthedocs-examples/awesome-read-the-docs>`_
+and help us discover more awesome projects.
+
+
+Read the Docs for Business
+--------------------------
+
+Considering using Read the Docs for an organization?
+
+.. raw:: html
+
+   <a href="https://about.readthedocs.com/" style="background-color: #409cff; border-radius: 3px; color: #ffffff; display: block; margin: 30px auto; font-size: 18px; font-weight: 700; line-height: 24px; padding: 15px 0 15px 0; text-align: center; text-decoration: none; width: 238px;">
+     Discover all the features ✨️
+   </a>
```

Comment:
Any feedback on this specific proposal or ideas for a different "call to action" button?

@humitos suggested at the last newsletter that we start having a large button. I didn't find anything in the content to immediately connect the message to, so this is a bit generic.

Reply:
Great! I'll make a note for the next newsletter to consider more "call to action" ideas.